airflow/include/tasks/extract/airflow_docs.py

```diff
@@ -1,36 +1,13 @@
 from __future__ import annotations
 
-import re
-import urllib.parse
-
 import pandas as pd
-import requests
-from bs4 import BeautifulSoup
-from weaviate.util import generate_uuid5
 
-from include.tasks.extract.utils.html_helpers import get_all_links
+from include.tasks.extract.utils.html_utils import get_internal_links
 
 
 def extract_airflow_docs(docs_base_url: str) -> list[pd.DataFrame]:
     """
-    This task scrapes docs from the Airflow website and returns a list of pandas dataframes. Return
-    type is a list in order to map to upstream dynamic tasks.  The code recursively generates a list
-    of html files relative to 'docs_base_url' and then extracts each as text.
-
-    Note: Only the (class_: body, role: main) tag and children are extracted.
-
-    Note: This code will also pickup https://airflow.apache.org/howto/*
-    which are also referenced in the docs pages.  These are useful for Ask Astro and also they are relatively few
-    pages so we leave them in.
-
-    param docs_base_url: Base URL to start extract.
-    type docs_base_url: str
-
-    The returned data includes the following fields:
-    'docSource': 'apache/airflow/docs'
-    'docLink': URL for the page
-    'content': HTML content of the page
-    'sha': A UUID from the other fields
+    This task return all internal url for Airflow docs
```

Comment:
If you look at diff the previous doc string looks better. We are missing many context for user about what we are doing.